lib/zold/commands/node.rb

```diff
@@ -171,6 +174,13 @@ def run(args = [])
       Front.set(:dump_errors, opts['dump-errors'])
       Front.set(:port, opts['bind-port'])
       Front.set(:reboot, !opts['never-reboot'])
+      node_alias = opts[:alias] || address
+      unless node_alias.eql?(address)
+        re = Regexp.new(/^[a-z0-9]{4,16}$/)
+        puts re.match(node_alias)
```

Comment:
@emilianodellacasa typo?

Reply:
It's for validating the alias, why should I remove it?

Reply:
@emilianodellacasa we don't use `puts`, we use logging instead